test/test_transforms.py

```diff
@@ -462,6 +462,16 @@ def test_resize_size_equals_small_edge_size(height, width):
     result = t(img)
     assert max(result.size) == max_size
 
+@pytest.mark.parametrize("height, width", [(28, 28)])
+def test_resize_equal_input_output_sizes(height, width):
+    # Regression test for https://github.com/pytorch/vision/issues/7518
+    img = Image.new("RGB", size=(width, height))
+
+    t = transforms.Resize(((height, width)), antialias=True)
+    result = t(img)
+    print(id(result),id(img))
+    assert id(result) == id(img)
```

Comment:
`id(obj1) == id(obj2)` is the same as `obj1 is obj2`:

```suggestion
    assert result is img
```